css/style.css

```diff
@@ -738,8 +741,20 @@ button.button-inline:hover {
 
 #boardlist td .app-popover-menu-utils {
 	float: right;
+	width: 30px;
+	display: inline;
+	position: relative;
+}
+
+#boardlist td .app-popover-menu-utils button {
+	opacity: 0.3;
```

Comment:
I don't really see the point in using 4 different opacity levels. Difference between 0.3 and 0.5 or 0.5 and 0.7 is barely noticable and it doesn't give away any additional message to the user.

I'm working on cleaning up opacity definitions in nextcloud/server and the idea is to use 0.5 and 1 for most of the situations, and I think this is a perfect example. Triple-dot button should  have 50% opacity normally and 100% when hovered/focused/selected wherever it is used.